### PR TITLE
Add supported devices to Refoss integration

### PR DESCRIPTION
**Pull request by ashionky**
Add description of supported device types

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/refoss.markdown [hunk 1]
```diff
@@ -26,6 +26,10 @@ Integrate Refoss devices into Home Assistant.
 - The integration will occupy port: 9989.    
 {% include integrations/config_flow.md %}
 
+### Supported device model
+
+- r10
```
Comment (c0ffeeca7) [inline source/_integrations/refoss.markdown]:
```suggestion
- Refoss Smart Wi-Fi Switch, R10
```
would it make sense here to use the full product name? That's how I found it on their website, but please correct me if I got it wrong.

Comment (ashionky) [reply source/_integrations/refoss.markdown:31]: The device model is r10，   Refoss Smart Wi-Fi Switch   is just  a classification.

Comment (ashionky) [reply source/_integrations/refoss.markdown]: You're right,    users can better understand

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @ashionky :+1:

**Pull request merged**